lib/schemapack.js

```diff
@@ -194,12 +194,18 @@ class SchemaPack {
     }
     // if the schema is validated, return the meta data as required.
     else if (this.validated) {
+      let name = _.get(this.openapi, 'info.title');
+
+      if (_.isEmpty(name) || !_.isString(name)) {
+        name = COLLECTION_NAME;
+      }
+
```

Comment:
The same code snippet, from line `197` to line `202`, is repeated at multiple locations. Is it possible to create a helper function to eliminate this repetition? By doing so, if there are any future changes, we won't have to modify it in multiple places, reducing the chance of inconsistencies.